src/stackdriver.js

```diff
@@ -23,12 +23,27 @@ function _levelToSeverity (level) {
   return 'default'
 }
 
+const defaultKeys = {
+  httpRequest: 'httpRequest',
+  trace: 'trace'
+}
```

Comment:
Can you make the props of defaultKeys optional like:
`const defaultKeys = { httpRequest: 'httpRequest' }`
and then when adding them to `entry.meta...` loop through keys to see if any exist like `Object.keys`

This way we can keep ik backwards compatible and not add anything  extra if not specifically asked for.

Reply:
This should be resolved in 2352d3d , I've removed `trace` from the defaults